lib/config/app_config.dart

```diff
@@ -74,6 +74,8 @@ abstract class AppConfig {
   static const int thumbnailQuality = 70;
   static const int blurHashSize = 32;
   static const int imageQuality = 50;
+  static const String iOSKeychainSharingId =
+      'KUT463DS29.com.linagora.ios.twake.shared';
```

Comment:
How about if it runs on debug? How to make easier for folked branch? How about to add also in config file?